packages/editor/src/client/index.tsx

```diff
@@ -129,6 +129,8 @@ const onDOMContentLoaded = async () => {
   )
 }
 
+initI18N()
```

Comment:
Glad you see the inits in _definitions_ as problem. 👍

Reply:
Yeah I like to keep side effects to a minimum 😄